Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -23,10 +24,10 @@ private static void Test(string args, int expected)
         public static int FindDigit(string equation)
         {
             // Add your code here.
-            int result=0;
-            char[] c1 = new char[2]{'*','='};
-            String[] sar=equation.Split(c1);
-            int finder=0;
+            int result=0; // in this variable we will returrn the result
+            char[] c1 = new char[2]{'*','='}; //character array for splliting the string into variable parts
+            String[] sar=equation.Split(c1); //split the equation to A, B and C strings
+            int finder=0; // will tell you which string have ? in it.
             for(int i=0;i<sar.Length;i++)  
             {
                 if(sar[i].Contains("?"))

```

Comment:
you are comparing character. Use sar[i] == '?'

Reply:
mam sar[i] is itself is a string  that contains a number like 12?4...
sar[] is an array of strings.